.github/workflows/release.yaml

```diff
@@ -21,10 +21,8 @@ jobs:
       run: |
         sudo apt-add-repository ppa:ubuntugis/ubuntugis-unstable
         sudo apt-get update
```

Comment:
You can actually get rid of this whole apt section now

Reply:
Done